bip-0003.md

```diff
@@ -298,7 +302,7 @@ Transitions involving the Closed state are:
 ##### Draft ↦ Closed
 
 BIP authors may decide on their own to change their BIP’s status from Draft to Closed. If a Draft BIP stops making
-progress, sees accumulated feedback unaddressed, or otherwise appears stalled for a year, the community may move the BIP
+progress, sees accumulated feedback unaddressed, or otherwise appears stalled for a year, anyone may move the BIP
```

Comment:
Perhaps

s/anyone may move the BIP to Closed/anyone may propose the BIP status be updated to Closed/